index.html

```diff
@@ -117,20 +117,30 @@ <h2>API</h2>
 import * as parsel from "./parsel.js";
 self.parsel = parsel; // so that people can experiment in the console
 
+const url = new URL(location);
+if(url.searchParams.get("selector")) {
+    let selector = url.searchParams.get("selector");
+    selectorText.value = selector || selectorText.value; // set default value if the selector is empty
```

Comment:
If the selector is empty, this conditional won't run at all (`""` is falsy), so I don't think we need to account for that.

Reply:
If the selector is empty, it shouldn't work at all. What do you mean exactly?

Reply:
Do you mean that line 123 is unnecessary?

Reply:
If the `selector` param in the URL is empty, the `if` will be skipped anyway, so there's no need to do `|| selectorText.value`.